tests/test_models.py

```diff
@@ -785,6 +801,7 @@ class Model(models.DomainModel):
 
     def test_models_collection_extending(self):
         """Test model's collection extending."""
+
```

Comment:
I'm not sure regarding those indents... Pydocstyle (pep257) restricts putting of blank lines after function docstrings:

```
786 error D202: No blank lines allowed after function docstring (found 1) [python/pydocstyle]
```

It makes sense at some point, cause it prevents us from doing something like this:

```python
def f():
    """Some comment."""
    
    return True